include/dxc/Support/HLSLOptimizationOptions.h

```diff
@@ -0,0 +1,24 @@
+///////////////////////////////////////////////////////////////////////////////
+//                                                                           //
+// HLSLOptions.h                                                             //
+// Copyright (C) Microsoft Corporation. All rights reserved.                 //
+// This file is distributed under the University of Illinois Open Source     //
+// License. See LICENSE.TXT for details.                                     //
+//                                                                           //
+// Option defined related to optimization customization.                     //
+//                                                                           //
+///////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#ifndef LLVM_HLSL_OPTIMIZATION_OPTIONS_H
```

Comment:
Already have #pragma once, do we still want #ifdef here?

Reply:
Probably not! but the file I copied it from had both and it looks pretty empty anyway 😜